src/python/grpcio_tests/tests/interop/methods.py

```diff
@@ -44,25 +44,48 @@
 from src.proto.grpc.testing import messages_pb2
 from src.proto.grpc.testing import test_pb2
 
+_INITIAL_METADATA_KEY = "x-grpc-test-echo-initial"
+_TRAILING_METADATA_KEY = "x-grpc-test-echo-trailing-bin"
 
 class TestService(test_pb2.TestServiceServicer):
 
+  # If the request came with metadata in the context, this
+  # helper function will copy it to the returned metadata.
+  def MaybeEchoMetaData(self, context):
```

Comment:
No use of `self` - why is this an instance-scope method rather than a module-scope helper function?